build.gradle.kts

```diff
@@ -87,6 +101,7 @@ configure<LicenseExtension> {
     header = project.rootProject.file("gradle/licenseHeader.txt")
     mapping(kotlin.collections.mapOf("kt" to "SLASHSTAR_STYLE", "java" to "SLASHSTAR_STYLE"))
     strictCheck = true
+    exclude("**/UseStringIsEmpty*.java")
```

Comment:
Should we move the files into a specific `src/test/resources` subdirectory and then exclude all of the files in that subdirectory?

Reply:
Good suggestion; moved those into `src/test/resources/recipes` and updated the license header exclude.